ms-inventory/res/configuration.yaml

```diff
@@ -0,0 +1,18 @@
+# Copyright © 2023 Intel Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+
+Writable:
+  LogLevel: INFO
+
+Service:
+  Host: localhost
+  Port: 48095
+  StartupMsg: This microservice exposes a CRUD interface for an inventory of items in a Automated Checkout
```

Comment:
```suggestion
  StartupMsg: This microservice exposes a CRUD interface for an inventory of items in an Automated Checkout System
```

Reply:
@seanohair22 , @ejlee3 , @brian-intel , we should change all occurrences of `Automated Checkout` to `Automated Vending` in the code and messages.